modules/scheduler/gke-cluster/main.tf

```diff
@@ -59,6 +97,10 @@ resource "google_container_cluster" "gke_cluster" {
 
   network    = var.network_id
   subnetwork = var.subnetwork_self_link
+  # Note: Though the default value of VPC_NATIVE is sufficient to enable IP Aliasing,
```

Comment:
My counter argument which you may not agree to is, if GKE decides that default mode is not `VPC_NATIVE` but something else based on the other variables that are passed in, maybe we do want to default to that ?